recipes/geoplot/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "geoplot" %}
+{% set version = "0.0.2" %}
+{% set md5 = "4c27bfaf5da2bcb75623bba3a9539ad5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/3a/40/45951c006fdd7f08e33230e7f70809869788f93a7500816b7ce952dfd814/{{ name }}-{{ version }}.tar.gz
```

Comment:
This URL was generated after I ran `conda skeleton pypi geoplot`, as per the instructions [here](https://conda.io/docs/build_tutorials/pkgs.html), to initialize this file. It's not nearly as pretty as the URL in the [example file](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18), though. Why is this? Is it because I'm uploading to PyPi using the [legacy route](http://peterdowns.com/posts/first-time-with-pypi.html) instead of via Warehouse? Pointers appreciated.